tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
+  It is also used for the `rootProject.name` value in the project's `settings.gradle` file.
+* _Version_ is used for the `project.version` value in the `build.gradle` file. 
+
+Click _Next_ to continue. 
+
+### Gradle Settings Screen
+The third screen prompts for Gradle-specific settings. 
+All of these settings can be changed once the project is created via **Settings \| Build, Execution, Deployment \| Build Tools \| Gradle**. 
+* Select whether to [_Use auto-import_](https://www.jetbrains.com/help/idea/gradle-settings.html).
+* Select _Create directories for empty content roots automatically_.
+  This selection will add a `src` directory to your project.
+* Choose how to [_Group modules_](https://www.jetbrains.com/help/idea/creating-and-managing-modules.html#grouping-modules), either by IntelliJ IDEA logical grouping or by Qualified Names. 
+  For new projects grouping modules by qualified names is recommended.
+* Create a separate module per Gradle [`SourceSet`](https://docs.gradle.org/current/userguide/java_plugin.html#source_sets). 
+  For a new project, this option results in `main` and `test` modules.
+* It’s recommended to select the [_Use default gradle wrapper_](https://docs.gradle.org/current/userguide/gradle_wrapper.html) option.
+  That way IntelliJ IDEA installs everything you need to run Gradle tasks. 
+* Skip using a local Gradle distribution, at least initially.
+  The Gradle plugin automatically downloads the preferred version of Gradle.
+  A local Gradle distribution can be used, but ensure it meets the version requirements of the [Gradle plugin](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md). 
+* Finally, specify a JVM for Gradle.
+  It can be the _Project JDK_, which was specified as the [_Project SDK_](#new-project-configuration-screen) on the first screen. 
+  
+Click _Next_ to continue. 
 
 ![Verify the JVM is the correct version](img/step3_gradle_config.png){:width="800px"}
 
-### 1.3 Configuring a Gradle Plugin Project
-Support for Gradle-based plugin projects is provided by the IntelliJ Platform `gradle-intellij-plugin`. 
+### Project Name and Location Screen
+The final Wizard screen is for setting the [project name and location](https://www.jetbrains.com/help/idea/project-name-and-location.html#Project_Name_and_Location.xml).
+The _Project name_ is pre-filled with the [Artifact ID](#project-naming-screen).
+
+Note the choice of _Project format_ under _More Settings_ is somewhat superfluous.
+Although an `.idea` directory or `.ipr` file is generated as the project is created and built, it is Gradle and the IntelliJ IDEA Gradle plugin that control many aspects of the project.  
+
+Click _Finish_.
+
+### Components of a Wizard-Generated Gradle IntelliJ Platform Plugin
+The New Project Wizard produces the project layout shown below:
+* The default IntelliJ Platform `build.gradle` file.
+  The contents are further discussed below.
```

Comment:
can we just mention "all necessary files for running Gradle" and link to Gradle docs instead?

Reply:
That would be sufficient for experienced Gradle users, I'm not so sure about users new to both Gradle and IntelliJ Platform plugin development. 

How about the following changes:
* Remove bullet items:
  * The shell scripts...
  * The structure under...
* Add bullet item:
  * Note addition of META-INF directory under resources in SourceSet.